src/main/java/com/google/opensesame/projects/ProjectPreview.java

```diff
@@ -0,0 +1,102 @@
+package com.google.opensesame.projects;
+
+import com.google.opensesame.github.GitHubGetter;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GitHub;
+
+public class ProjectPreview {
+  /**
+   * Creates a ProjectPreview object from a set of project properties.
+   *
+   * @param properties The properties of a project entity. This can come directly from a datastore
+   *     entity, as datastore entities inherit a getProperty() method from PropertyContainer:
+   *     https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/Entity
+   * @return Returns the created ProjectPreview.
+   * @throws IOException Throws when there is an error communicating with the GitHub API.
+   */
+  public static ProjectPreview fromProperties(Map<String, Object> properties) throws IOException {
+    GitHub gitHub = GitHubGetter.getGitHub();
+    GHRepository repository = gitHub.getRepository((String) properties.get("repositoryName"));
+
+    return fromPropertiesAndRepository(properties, repository);
+  }
+
+  /**
+   * Creates a ProjectPreview object from a set of project properties and it's associated GitHub
+   * repository.
+   *
+   * @param properties The properties of a project. This will most likely come from a datastore
```

Comment:
Properties is the same in both `fromProperties` and `fromPropertiesAndRepository` so it should have the same docstring.
```suggestion
   * @param properties The properties of a project entity. This can come directly from a datastore
   *     entity, as datastore entities inherit a getProperty() method from PropertyContainer:
   *     https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/Entity
```

Reply:
Good catch. Will apply your changes.